src/components/repo-list/RepoList.js

```diff
@@ -57,7 +57,12 @@ export const RepoList = ({ repos, user }) => {
 
   const SelectFilter = () => (
     <Flex justifyContent="flex-end" mb="12px">
-      <Select fontSize="15px" onChange={onFilterChange} w="160px">
+      <Select
+        fontSize="15px"
+        onChange={onFilterChange}
+        value={repoFilter}
```

Comment:
adds value prop

Reply:
small note, I dont think this Select is actually a filter in terms of what it does. maybe later we can rename